### DISCUSSION TITLE
Go back button implementation

### DISCUSSION THREAD

**Discussion by Aqamarine228** [MohmmedAshraf/laravel-translations#8] [Ideas]
The UI of translations has beautiful design and provides good UX. However when used as an addition to admin panel there is one UI detail missing which makes it unable  to provide slight transition between translations UI and rest of the admin panel. It would be great when there would be a go back button, which path could be edited in translations config file.
### Example as it's implemented in Laratrust UI panel:
<img width="1225" alt="Снимок экрана 2023-01-09 в 12 17 03" src="https://user-images.githubusercontent.com/62146368/211296472-1749cb12-79db-4db7-aa81-4e23dbd4acba.png">
<img width="628" alt="image" src="https://user-images.githubusercontent.com/62146368/211296533-d6851be3-2296-4d06-bf9c-947bdc431002.png">

**Comment by MohmmedAshraf** [comment]
Hi @Aqamarine228 ,

Thank you for your feedback, This is a good idea and I will consider adding this feature in a future update. I appreciate your input and will take it into consideration as we continue to improve the translations UI.

Best regards,
